JSDemos/Demos/DataGrid/BatchUpdateRequest/description.md

```diff
@@ -15,3 +15,5 @@ Once changes are saved, call the [refresh(changesOnly)](/Documentation/ApiRefere
 
 4. **Reset edit state**           
 Use the [cancelEditData()](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#cancelEditData) method to clear pending changes.
+
+Related demos: [Web API Service](/Demos/WidgetsGallery/Demo/DataGrid/WebAPIService/), [Batch Editing](/Demos/WidgetsGallery/Demo/DataGrid/BatchEditing/).
```

Comment:
We need to clarify why we consider these demos related. What additional information will readers get from them?